components/developer/gcc-7/Makefile

```diff
@@ -46,7 +46,7 @@ include $(WS_MAKE_RULES)/gcc-component.mk
 
 CONFIGURE_OPTIONS+= --enable-plugins
 CONFIGURE_OPTIONS+= --enable-objc-gc
-CONFIGURE_OPTIONS+= --enable-initfini-array
+CONFIGURE_OPTIONS.i386+= --enable-initfini-array
```

Comment:
This doesn't work as expected

Reply:
no?

make-rules/gcc-component.mk:CONFIGURE_OPTIONS+= $(CONFIGURE_OPTIONS.$(MACH))

Reply:
Sorry, was wrong.